src/features/nft/addNFT/previewStep/PreviewStep.tsx

```diff
@@ -59,57 +61,80 @@ export default function PreviewStep({
       titleClass='mb-3'
       subtitle='Description'
       step={3}
+      leftColumnWidth={image.maxWidth}
       leftColumnContent={
-        <div className='relative'>
+        <div className='relative z-10'>
+          {imageElement && (
+            <Magnification
+              image={image}
+              imageElement={imageElement}
+              isLossLess={isLossLess}
+            />
+          )}
           <FullScreenButton onClick={toggleFullScreen} />
           <Tooltip text='Crop thumbnail'>
             {ref => (
               <button
                 ref={ref}
-                className='absolute bottom-3.5 left-3.5 w-10 h-10 text-white flex-center rounded-full bg-gray-2d bg-opacity-50'
+                className='absolute z-10 bottom-3.5 left-3.5 w-10 h-10 text-white flex-center rounded-full bg-gray-2d bg-opacity-50'
                 onClick={toggleCropping}
               >
                 <Crop size={18} />
               </button>
             )}
           </Tooltip>
-          <img src={image} className='rounded' />
+          <img
+            ref={setImageElement}
+            src={image.url}
+            className={`rounded ${style.zoomInCursor}`}
+          />
         </div>
       }
       rightColumnContent={
-        <div className='space-y-8'>
-          <div>
-            <div className='font-medium text-gray-71 mb-3'>Image size</div>
-            <div className='flex'>
-              <div className='text-gray-2d text-sm font-extrabold w-20'>
-                50 Mb
+        <div>
+          <div className='flex items-start mb-8'>
+            <div className='font-medium text-gray-4a mr-5'>Image size</div>
+            <div className='text-gray-2d text-sm font-extrabold mr-3 relative top-[3px]'>
+              50 Mb
+            </div>
+            <div className='flex-grow'>
+              <div className='bg-gray-e4 bg-opacity-50 rounded h-2 relative my-2'>
+                <div
+                  className='absolute top-0 left-0 bottom-0 rounded bg-green-62'
+                  style={{ width: '65%' }}
```

Comment:
This percentage is mock data. In future it will be replaced with value from API.
To use tailwind here, we should add to tailwind "width" config full sequence from 1 to 100. it seems a bit excessive.